index.html

```diff
@@ -121,6 +121,36 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ Harshabardhana Parida Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Harshabardhana Parida</p>
+          <p class="contact">
+            <i class="fab fa-linkedin"></i>
+            <a href="www.linkedin.com/in/harshaparida" target="_blank">harshaparida</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/harshaparida" target="_blank">harshaparida</a>
+          </p>
+          <p class="about">
+            Hi! I'm a Student developer, and Machine Learning enthusiast. New to open-source. Feel free to contact me!
+          </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.w3schools.com/https://www.w3schools.com/" target="_blank" title="First Resource">W3Schools</a>
```

Comment:
The hyper link seems to be pasted twice inside the `href` attribute. Kindly rectify that :)

Reply:
Thank you for letting me know. I have rectified the hyperlink.
![edit](https://github.com/Syknapse/Contribute-To-This-Project/assets/119295005/0c186aef-385e-4f47-b3e5-0d7ba1e72af8)